.github/workflows/controller-test.yml

```
@@ -30,8 +30,16 @@ jobs:
     name: Run on Ubuntu
     runs-on: ubuntu-latest
     steps:
+      - name: Generate tag
+        shell: bash
+        id: tags
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:3:22: Double quote to prevent globbing and word splitting

(shellcheck)